docs/content/concepts/assets/asset-observations.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Asset Observations | Dagster
+description: Dagster provides functionality to record metadata about assets.
+---
+
+# Asset Observations
+
+An asset observation is an event that records metadata about a given asset. Unlike asset materializations, asset observations do not signify that an asset has been mutated.
+
+## Relevant APIs
+
+| Name                                   | Description                                                          |
+| -------------------------------------- | -------------------------------------------------------------------- |
+| <PyObject object="AssetObservation" /> | Dagster event indicating that an asset's metadata has been recorded. |
+| <PyObject object="AssetKey" />         | A unique identifier for a particular external asset.                 |
+
+## Overview
+
+<PyObject object="AssetObservation" /> events are used record metadata in Dagster
+about a given asset. Asset observation events can be yielded at runtime within ops
+and assets. An asset must be defined using the <PyObject
+object="asset"
+decorator
+/> decorator or have existing materializations in order for its observations to be
+displayed.
+
+## Yielding an AssetObservation from an Asset
+
+To make Dagster aware that we have recorded metadata about an asset, we can yield an <PyObject object="AssetObservation" /> event:
+
+```python file=/concepts/assets/observations.py startafter=start_observation_asset_marker_0 endbefore=end_observation_asset_marker_0
+from dagster import AssetObservation
+
+
+@op
+def observation_op():
+    df = read_df()
+    remote_storage_path = persist_to_storage(df)
+    yield AssetObservation(asset_key="observation_asset", metadata={"num_rows": len(df)})
+    yield AssetMaterialization(
+        asset_key="observation_asset", description="Persisted result to storage"
+    )
+    yield Output(5)
+```
+
+We should now see an observation event in the event log when we execute this asset.
+
+<Image
+alt="asset-observation"
```

Comment:
Are we able to add an entry to screenshots.yaml for generating this screenshot?

Reply:
added